### DISCUSSION TITLE
Documentation required on how to integrate Quick Actions against variations

### DISCUSSION THREAD

**Discussion by leewillis77** [woocommerce/woocommerce#43135] [WooCommerce New Product Editor] [answered]
The variations section of the new product editor allows "quick actions" to be carried out against selected variations

![Screenshot 2023-12-27 at 16 11 49](https://github.com/woocommerce/woocommerce/assets/1097338/3f12ec7f-f17e-4a3a-bf2f-4ba4749a6ce3)

These are called "Bulk actions" in the current editor, and can be added to using the `woocommerce_variable_product_bulk_edit_actions` and `woocommerce_bulk_edit_variations` filters.

I can't find any documentation on how to register "quick actions" in the new editor.

**Comment by mattsherman** [comment, marked as answer]
Hi @leewillis77 !

We will actually be implementing support for this in the new product editor soon. You can follow the issue here:

- https://github.com/woocommerce/woocommerce/issues/42737

**Comment by nerrad** [comment]
It looks like the usecase described in this discussion has been addressed by the PR that @mattsherman  mentioned so this discussion can be closed.

**Comment by leewillis77** [comment]
The current implementation as per that PR isn't really suitable to 3rd party extensions, see the comments re: grouping on #43441. However, fine to leave this closed to avoid duplication if that makes sense.

**Comment by leewillis77** [comment]
Also, there is (afaik) no documentation on this which was part of the original request.

**Comment by nerrad** [comment]
> Also, there is (afaik) no documentation on this which was part of the original request.

Point heard! @louwie17 can you make sure there's an issue to capture the documentation need outlined in this discussion thread?
